packages/flutter_tools/lib/src/isolated/devfs_web.dart

```diff
@@ -866,7 +897,12 @@ class WebDevFS implements DevFS {
       generator.addFileSystemRoot(outputDirectoryPath);
       final String entrypoint = globals.fs.path.basename(mainFile.path);
       webAssetServer.writeBytes(entrypoint, mainFile.readAsBytesSync());
-      webAssetServer.writeBytes('require.js', requireJS.readAsBytesSync());
+      if (ddcModuleSystem) {
+        webAssetServer.writeBytes(
+            'ddc_module_loader.js', ddcModuleLoaderJS.readAsBytesSync());
```

Comment:
We're still on time of giving this module loader a catchy name! `need.js`? :P

Reply:
Haha, ddc_module_loader.js is just consistent with what it does (load modules, which are bundles of Dart libs).

Reply:
![](https://media1.tenor.com/m/RpxxkrGJ5VIAAAAC/thumbs-up-engineer-gaming.gif)